tests/unitTests/app/models/GradeableComponentTester.php

```diff
@@ -12,6 +13,12 @@ public function setUp() {
         $this->core = $this->createMock(Core::class);
     }
     
+    protected function createMockUser($id) {
+        $return = $this->createMock(User::class);
+        $return->method("getId")->willReturn($id);
+        return $return;
+    }
```

Comment:
Make this class extend from [BaseUnitTest.php](https://github.com/Submitty/Submitty/blob/master/tests/unitTests/BaseUnitTest.php) and use [createMockModel](https://github.com/Submitty/Submitty/blob/master/tests/unitTests/BaseUnitTest.php#L93-L115) function or else all tests that use this method are just going to issue a warning and not properly test.